src/lib/chat-ui.js

```diff
@@ -340,18 +341,23 @@ const ChatUi = {
   addOptions() {
     const element = this.getLastMessageElement('.assistant');
     const answerConfig = getAnswerConfig(this.answersFromStream);
-
     const answersContainer = document.createElement('div');
     answersContainer.classList.add('answers-container');
     [...answerConfig.list].forEach((answer) => {
       const optionElement = document.createElement('div');
       optionElement.textContent = answer.content;
-      actionService.handleAction(optionElement, answer.actions);
+      actionService.handleAction(optionElement, answer.actions, answersContainer);
       optionElement.addEventListener('click', this[answerConfig.answersType].bind(this));
       answersContainer.appendChild(optionElement);
     });
     this.answersFromStream = '';
-    element.appendChild(answersContainer);
+
+    [...answerConfig.list].forEach((btn) => {
+      if (!btn.actions.includes('10')) {
```

Comment:
what is this magical number 10?

Reply:
#10# that how the initial buttons will be marked if we want them out of the chat bubble

Reply:
than set this as some type of constant like STICKY_TAGGED so it is readable